src/main/resources/view/DarkTheme.css

```diff
@@ -127,11 +127,22 @@
 }
 
 .cell_small_label {
-    -fx-font-family: "Segoe UI";
+    -fx-font-family: "Segoe UI Semibold";
     -fx-font-size: 13px;
     -fx-text-fill: #010504;
 }
 
+.currently_displaying {
+    -fx-font-family: "Segoe UI Semibold";
```

Comment:
Would it be possible to change the font family to this instead?
```
* {
   -fx-font-family: 'serif'
}
```

Rationale: My 2015 mac is unable to read the Segoe UI unfortunately.

Reply:
Changed the font to sans-serif